odoo_sh/getting_started/branches.rst

```diff
@@ -0,0 +1,310 @@
+Be careful though: If these actions perform changes on third-party services (FTP servers, email servers, ...)
+used by your production database as well,
+you might make unwanted changes in your production.
+
+The databases created for staging branches are meant to live at least two weeks.
+After that, they can be garbage collected automatically.
+If you make configuration changes or view changes in these branches, make sure to document them or write them directly
+in the modules of the branch, using XML data files overriding the default configuration or views.
+
+The unit tests are not performed as, in Odoo, they currently relies on the demo data, which are not loaded in the
+production database. In the future, if Odoo supports to run the unit tests without the demo data,
+Odoo.sh will then consider running the tests on staging databases.
+
+
+Development
+-----------
+Development branches creates new databases using the demo data and running the unit tests.
+The modules installed and tested by default are the ones included in your branches.
+You can change this list of modules to install in your projects settings.
+
+When you push a new commit in one of these branches,
+a new server is started, with a database created from scratch and the new revision of the branch.
+The demo data is loaded, and the unit tests are performed.
+
+You can therefore verify your changes do not break any of the unit tests,
+and therefore any of the features tested thanks to them.
+
+As for staging branches, the emails are not sent: They are intercepted by the mailcatcher.
+
+The databases created for development branches are meant to live at least three days.
+After that, they can be garbage collected automatically.
+
+Merging your branches
+---------------------
+You can merge your branches easily by drag and dropping them on each other.
+
+.. image:: ./media/interface-branches-merge.png
+   :align: center
+
+When you want to test the changes of your development branches with the production data,
+you can either:
+
+* merge the development branch into your staging branch, by drag and dropping it onto the desired staging branch,
+* drag and dropping the development branch on the staging section title, to make it become a staging branch.
+
+When your latest changes are ready for production,
+you can drag and drop your staging branch onto your production branch
+to merge and deploy in production your newest revisions.
+
+If you are bold enough,
+you can merge your development branches into your production branch as well.
+It just means you skip the validation of your changes with the production data through a staging branch.
+
+You can merge your development branches into each other, and your staging branches into each other.
+
+Of course, you can also use git directly to merge your branches.
+Odoo.sh will be notified new revisions have been pushed in your branches.
```

Comment:
Better = Odoo.sh will be notified when new revisions have been pushed in your branches.